cloudrun-malware-scanner/cloudbuild.yaml

```diff
@@ -0,0 +1,25 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/clamav-scanner:latest', '.']
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['push', 'gcr.io/$PROJECT_ID/clamav-scanner:latest']
+- name: 'gcr.io/google.com/cloudsdktool/cloud-sdk'
+  entrypoint: 'gcloud'
+  args:
+  - 'run'
+  - 'deploy'
+  - 'clamav-scanner'
+  - '--image'
+  - 'gcr.io/$PROJECT_ID/clamav-scanner:latest'
+  - '--no-allow-unauthenticated'
+  - '--service-account'
+  - 'clamav-scanner@development-344423.iam.gserviceaccount.com'
+  - '--cpu-boost'
+  - '--region'
+  - 'us-central1'
+  - '--project'
+  - '$PROJECT_ID'
+  - '--set-env-vars'
+  - 'CVD_MIRROR_BUCKET=${_CVD_MIRROR_BUCKET}'
```

Comment:
so i would remove bucket env out of deploy step..
this step should only update the binary

Reply:
Done.